packages/gestalt/src/TableSortableHeaderCell.js

```diff
@@ -59,6 +59,34 @@ type Props = {|
   status: 'active' | 'inactive',
 |};
 
+function SortIcon({
+  align,
+  status,
+  sortOrder,
+  visibility,
+}: {|
+  align: 'start' | 'end',
+  status: 'active' | 'inactive',
+  sortOrder: 'asc' | 'desc',
+  visibility: 'visible' | 'hidden',
+|}) {
+  return (
+    <Box
+      marginStart={align === 'start' ? 2 : undefined}
+      marginEnd={align === 'end' ? 2 : undefined}
+      dangerouslySetInlineStyle={{
+        __style: { visibility },
+      }}
+    >
+      <Icon
+        accessibilityLabel=""
```

Comment:
Why is accessibilityLabel="" empty?

Shouldn't it be translated?

Reply:
Check video re accessility, I think a11y is broken. https://capture.dropbox.com/HB4Talz6UVIkYEXZ

Reply:
Agree, it's not perfect. It's been like that for a while, and it might be a separate item. I actually noticed empty accessbility labels on a couple of other icons as well like in SideNav btw

Also, I took a look at video.  But some of the w3 examples (e.g. mute) seem to behave similar:
https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/button/button.html

I think we need to do this two -fold:
1. Add role toggle button to TapArea
2. Put the label in the accessbility provider, but it should be dynamic, so I feel like it won't work there.